code/modules/jobs/job_types/wasteland.dm

```diff
@@ -1,1119 +1,1287 @@
+	uniform = /obj/item/clothing/under/f13/desert_ranger_scout
+	shoes = /obj/item/clothing/shoes/f13/cowboy
+	head = /obj/item/clothing/head/f13/cowboy
+	gloves = /obj/item/clothing/gloves/color/brown
+	l_hand = /obj/item/gun/ballistic/revolver/colt357
+	backpack_contents = list(
+		/obj/item/ammo_box/a357=2,
+		/obj/item/binoculars=1,
+		/obj/item/radio=1)
+
+/datum/job/wasteland/f13enforcer
+	name = "Den Mob Enforcer"
+	flag = F13ENFORCER
+	faction = "Wastelander"
+	social_faction = "Raiders"
+	total_positions = 6
+	spawn_positions = 6
+	description = "The mob rules in Yuma. A hitman for the Den's Boss, you are a highly loyal enforcer charged with keeping order among the outlaw groups inhabiting the Den."
+	supervisors = "The Boss."
+	selection_color = "#ff4747"
+	exp_requirements = 600
+	exp_type = EXP_TYPE_OUTLAW
+
+	outfit = /datum/outfit/job/wasteland/f13enforcer
+
+	access = list(ACCESS_DEN)
+	minimal_access = list(ACCESS_DEN)
+
+	loadout_options = list(
+		/datum/outfit/loadout/hitman,
+		/datum/outfit/loadout/bodyguard,
+		/datum/outfit/loadout/ronin,
+		/datum/outfit/loadout/ashigari,
+		/datum/outfit/loadout/soldat,
+        /datum/outfit/loadout/soldatscouter,
+		/datum/outfit/loadout/sovietmedic,
+		/datum/outfit/loadout/colombianhitman,
+		/datum/outfit/loadout/narcos,
+		/datum/outfit/loadout/shotgunner
+		)
+
+
+/datum/outfit/job/wasteland/f13enforcer
+	name = "Den Mob Enforcer"
+	jobtype = /datum/job/wasteland/f13enforcer
+
+	id = /obj/item/card/id/dentattoo
+	belt = /obj/item/storage/belt/military/assault
+	shoes = /obj/item/clothing/shoes/laceup
+	ears = /obj/item/radio/headset/headset_den
+	l_pocket = /obj/item/melee/onehanded/knife/switchblade
+	r_pocket = /obj/item/flashlight/seclite
+	uniform = /obj/item/clothing/under/suit/white
+	backpack =	/obj/item/storage/backpack/satchel
+	satchel =  /obj/item/storage/backpack/satchel
+	gloves =  /obj/item/clothing/gloves/color/white
+	head = /obj/item/clothing/head/beret/durathread
+	mask =  /obj/item/clothing/mask/bandana/durathread
+	backpack_contents = list(
+		/obj/item/reagent_containers/hypospray/medipen/stimpak=1, \
+		/obj/item/restraints/handcuffs=1, \
+		/obj/item/storage/bag/money/small/wastelander)
+
+/datum/outfit/job/wasteland/f13enforcer/pre_equip(mob/living/carbon/human/H)
+	..()
+	r_hand = /obj/item/book/granter/trait/selection
+
+/datum/outfit/job/wasteland/f13enforcer/post_equip(mob/living/carbon/human/H, visualsOnly = FALSE)
+	..()
+	if(visualsOnly)
+		return
+
+	if(!H.gang)
+		var/datum/gang/denmob/DM = GLOB.denmob
+		GLOB.all_gangs |= DM
+		DM.add_member(H)
+		H.gang = DM
+
+/datum/outfit/loadout/ronin
+	name = "Ronin"
+	r_hand = /obj/item/katana
+	suit = /obj/item/clothing/suit/armor/f13/combat/swat
+	shoes = /obj/item/clothing/shoes/laceup
+	belt = /obj/item/storage/belt/military/assault
+	ears = /obj/item/radio/headset/headset_den
+	id = /obj/item/card/id/dentattoo
+	backpack_contents = list(
+						 /obj/item/gun/ballistic/automatic/smg/mp5,
+						)

```

Comment:
Line 842, `/obj/item/gun/ballistic/automatic/smg/mp5,` has been indented with a mix of tabs and spaces.

Reply:
This is also one of the reasons your compilation is failing.

Reply:
Gimme a minute.

Reply:
I fixed it. It doesn't show up anymore.

Reply:
I have deleted the file as you wanted, but it seems i cannot remove it for some strange reason. It is almost 3 AM right now, and the only failed check is the output attonation. The others, i'm not so sure as they don't tell the specifics.

Reply:
Keep in mind to do indentation with tabs not spaces (see list below), also now there's a few dozen duplicate definitions.

<details>
    <summary>wrong indentation, use tabs instead of spaces</summary>

```code/modules/jobs/job_types/wasteland.dm:        /datum/outfit/loadout/soldatscouter,
code/modules/jobs/job_types/wasteland.dm:    /obj/item/clothing/suit/jacket/leather,
code/modules/jobs/job_types/wasteland.dm:  /obj/item/ammo_box/magazine/uzim9mm/rockwell/,
code/modules/jobs/job_types/wasteland.dm:  name = "Soviet Medic"
code/modules/jobs/job_types/wasteland.dm:  jobtype = /datum/job/wasteland/sovietmedic
code/modules/jobs/job_types/wasteland.dm:  r_hand = /obj/item/gun/ballistic/revolver/colt6520
code/modules/jobs/job_types/wasteland.dm:  suit = /obj/item/clothing/suit/jacket/puffer/vest
code/modules/jobs/job_types/wasteland.dm:  shoes = /obj/item/clothing/shoes/sneakers/noslip
code/modules/jobs/job_types/wasteland.dm:  belt = /obj/item/storage/belt/military/assault
code/modules/jobs/job_types/wasteland.dm:  backpack = /obj/item/storage/backpack/satchel
code/modules/jobs/job_types/wasteland.dm:  ears = /obj/item/radio/headset/headset_den
code/modules/jobs/job_types/wasteland.dm:  id = /obj/item/card/id/dentattoo
code/modules/jobs/job_types/wasteland.dm:  backpack_contents = list(
code/modules/jobs/job_types/wasteland.dm:  name = "colombian Hitman"
code/modules/jobs/job_types/wasteland.dm:  jobtype = /datum/job/wasteland/colombianhitman
code/modules/jobs/job_types/wasteland.dm:  r_hand = /obj/item/gun/ballistic/automatic/pistol/beretta
code/modules/jobs/job_types/wasteland.dm:  suit = /obj/item/clothing/suit/armor/vest
code/modules/jobs/job_types/wasteland.dm:  shoes = /obj/item/clothing/shoes/winterboots
code/modules/jobs/job_types/wasteland.dm:  belt = /obj/item/storage/belt/military/assault
code/modules/jobs/job_types/wasteland.dm:  backpack =	/obj/item/storage/backpack/satchel
code/modules/jobs/job_types/wasteland.dm:  ears = /obj/item/radio/headset/headset_den
code/modules/jobs/job_types/wasteland.dm:  id = /obj/item/card/id/dentattoo
code/modules/jobs/job_types/wasteland.dm:  backpack_contents = list(
code/modules/jobs/job_types/wasteland.dm:        
code/modules/jobs/job_types/wasteland.dm:  name = "Narcos"
code/modules/jobs/job_types/wasteland.dm:  jobtype = /datum/job/wasteland/narcos
code/modules/jobs/job_types/wasteland.dm:  r_hand = /obj/item/gun/ballistic/automatic/service
code/modules/jobs/job_types/wasteland.dm:  suit = /obj/item/clothing/suit/armor/f13/leather_jacket/combat/riotpolice
code/modules/jobs/job_types/wasteland.dm:  shoes = /obj/item/clothing/shoes/jackboots
code/modules/jobs/job_types/wasteland.dm:  belt = /obj/item/storage/belt/military/assault
code/modules/jobs/job_types/wasteland.dm:  ears = /obj/item/radio/headset/headset_den
code/modules/jobs/job_types/wasteland.dm:  id = /obj/item/card/id/dentattoo
code/modules/jobs/job_types/wasteland.dm:  backpack_contents = list(
code/modules/jobs/job_types/wasteland.dm:  backpack_contents = list(
code/modules/jobs/job_types/wasteland.dm: name = "Kapitan"
code/modules/jobs/job_types/wasteland.dm: jobtype = /datum/job/wasteland/kapitan
code/modules/jobs/job_types/wasteland.dm: r_hand = /obj/item/gun/ballistic/automatic/assault_rifle
code/modules/jobs/job_types/wasteland.dm: suit = /obj/item/clothing/suit/armor/vest
code/modules/jobs/job_types/wasteland.dm: belt = /obj/item/storage/belt/military/assault
code/modules/jobs/job_types/wasteland.dm: backpack =	/obj/item/storage/backpack/satchel
code/modules/jobs/job_types/wasteland.dm: ears = /obj/item/radio/headset/headset_den
code/modules/jobs/job_types/wasteland.dm: id = /obj/item/card/id/dentattoo
code/modules/jobs/job_types/wasteland.dm: backpack_contents = list(
code/modules/jobs/job_types/wasteland.dm: /obj/item/clothing/hat/beret/red)```